playground/editors/ScriptableEditor.js

```diff
@@ -200,6 +200,8 @@ export class ScriptableEditor extends BaseNodeEditor {
 
 				disposeScene( editorOutputAdded );
```

Comment:
This name is a bit misleading. `editorOutputAdded` is not a scene so `disposeScene()` usually operates on a mesh. Hence, I had to introduce a separate `resetScene()`.